intern/remote/remote.py

```diff
@@ -312,4 +312,22 @@ def mesh(self, resource, resolution,
         mesh = self._mesh.create(
             volume, x_range, y_range, z_range, time_range, id_list, voxel_unit, voxel_size,
             simp_fact, max_simplification_error, normals)
-        return mesh
\ No newline at end of file
+        return mesh
+
+    def get_cuboids_from_id(self, resource, resolution, id):
+        """Get corners for cuboids that belong to a specific ID. 
+
+        All returned corners are cuboid aligned.
+
+        Args:
+            resource (intern.resource.Resource): Resource compatible with annotation operations.
+            resolution (int): 0 indicates native resolution.
+            id (int): Id of object of interest.
+
+        Returns:
+            (dict): {'cuboids': [[512, 512, 64], [0, 512, 32], [512, 512, 32], [0, 512, 48]]}
```

Comment:
Looks good, just update the returns comment to match the new comments for `get_cuboids_from_id()`.